bool handle_used
---
bool handle_candidate
visualization_msgs/MarkerArray handle_marker
